packages/ui/src/components/Menu/index.tsx

```diff
@@ -54,41 +47,37 @@ const MenuItem = ({ icon, href, disabled, onClick, children }: MenuItemProps) =>
 
   if (href) {
     return (
-      <HeadlessMenu.Item>
-        {({ active }) => (
-          <a
-            href={href}
-            onClick={event => {
-              event.preventDefault();
-              router.push(href);
-            }}
-            className={`${active ? 'bg-stone-100 dark:bg-stone-800' : ''} ${
-              disabled ? 'cursor-not-allowed opacity-50' : ''
-            } select-none text-sm text-stone-800 dark:text-stone-300 px-4 py-1 w-full rounded-md text-left flex gap-2 items-center`}
-          >
-            {icon}
-            {children}
-          </a>
-        )}
-      </HeadlessMenu.Item>
-    );
-  }
-
-  return (
-    <HeadlessMenu.Item>
-      {({ active }) => (
-        <button
-          type="button"
-          onClick={onClick}
-          className={`${active ? 'bg-stone-100 dark:bg-stone-800' : ''} ${
+      <RadixMenu.Item asChild>
+        <a
+          href={href}
+          onClick={event => {
+            event.preventDefault();
+            router.push(href);
+          }}
```

Comment:
The Headless.ui docs [address integration](https://headlessui.com/react/menu#integrating-with-next-js) with Next.js. Should I implement their suggestion here as well?

Reply:
Yes, we can 👍🏻